### PR TITLE
Add `aria-disabled` to slider root

### PR DESCRIPTION
**Pull request by abdel-17**
This PR adds an aria-disabled attributes to the Slider's root element.

PS: sorry for doing 2 PRs, the old branch got messed up, so I deleted it.

### CODE REVIEW

File: src/lib/builders/slider/create.ts [hunk 1]
```diff
@@ -55,6 +55,7 @@ export const createSlider = (props?: CreateSliderProps) => {
 		returned: ([$disabled, $orientation]) => {
 			return {
 				disabled: disabledAttr($disabled),
+				'aria-disabled': $disabled ? 'true' : 'false',
```
Comment (TGlide) [inline src/lib/builders/slider/create.ts:58]: I think we can use the `disabledAttr` helper. It's easier for styling, and according to MDN it's valid behaviour to remove it when not disabled. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-disabled#associated_roles

Comment (abdel-17) [reply src/lib/builders/slider/create.ts:58]: The problem with the helper is that it sets it to the boolean true, not the string, so Tailwind’s `aria-disabled:` doesn’t work. So, you’d need to change the helper or just make another helper for aria disabled.

Comment (TGlide) [reply src/lib/builders/slider/create.ts:58]: I see what you mean. In that case, `'aria-disabled': $disabled ? 'true' : undefined` should suffice I believe, right?

Comment (abdel-17) [reply src/lib/builders/slider/create.ts:58]: Yeah sure. That should support both use cases `[aria-disabled]` and `[aria-disabled="true"]`.

Comment (TGlide) [reply src/lib/builders/slider/create.ts:58]: Perfect then :)

Comment (TGlide) [reply src/lib/builders/slider/create.ts]: @abdel-17 it seems Svelte resolves `true` to `"true"` in this case

Comment (abdel-17) [reply src/lib/builders/slider/create.ts]: Strange. Looks like `aria-disabled` and `aria-disabled={true}` behave differently when it comes to DOM elements.

**Pull request merged**